gradle/dependencies.gradle

```diff
@@ -40,7 +40,11 @@ ext.deps = [
         postgres: "org.testcontainers:postgresql:${versions.testContainers}"
     ],
     valiktor: "org.valiktor:valiktor-core:0.12.0",
-    logback: "ch.qos.logback:logback-classic:1.2.3",
+    logs: [
+        logback: "ch.qos.logback:logback-classic:1.2.3",
+        logbackJsonEncoder: "net.logstash.logback:logstash-logback-encoder:6.4",
```

Comment:
`logbackJsonEncoder` uses jackson as JSON library, but I'm currently looking for an alternative to use the JSON library we want.